src/khoj/routers/api_chat.py

```diff
@@ -73,6 +73,48 @@
 from khoj.routers.email import send_query_feedback
 
 
+class FilterRequest(BaseModel):
+    filename: str
+    conversation_id: str
+
+
+@api_chat.get("/conversation/file-filters/{conversation_id}", response_class=Response)
+@requires(["authenticated"])
+def get_file_filter(request: Request, conversation_id: str) -> Response:
+    conversation = ConversationAdapters.get_conversation_by_user(
+        request.user.object, conversation_id=int(conversation_id)
+    )
+    file_filters = conversation.file_filters
```

Comment:
Likewise here, users might have deleted some files in between the time they selected the config and when this data is being returned. Would be good to filter those out from the list.